spec/calculator.spec.js

```diff
@@ -16,288 +16,304 @@
-      instance.addNumber(3);
+      instance.addNumber('3');
       instance.addPlus();
       instance.addMinus();
-      instance.addNumber(1);
+      instance.addNumber('1');
       instance.equal();
-      expect(instance.input()).to.equal(2);
+      expect(instance.input()).to.equal("2");
     });
 
-    it('It should show correct result even if user pressed an operation button twice', () => {
-      instance.addNumber(5);
+    it('It should show correct result even if user pressed an subtract operation button twice', () => {
+      instance.addNumber('5');
       instance.addMinus();
       instance.addMinus();
-      instance.addNumber(2);
+      instance.addNumber('2');
       instance.equal();
-      expect(instance.input()).to.equal(3);
-
-      instance.clear();
+      expect(instance.input()).to.equal("3");
+    });
 
-      instance.addNumber(2);
+    it('It should show correct result even if user pressed an add operation button twice', () => {
+      instance.addNumber('2');
       instance.addPlus();
       instance.addPlus();
-      instance.addNumber(3);
+      instance.addNumber('3');
       instance.equal();
-      expect(instance.input()).to.equal(5);
-
-      instance.clear();
+      expect(instance.input()).to.equal("5");
+    });
 
-      instance.addNumber(6);
+    it('It should show correct result even if user pressed an multiply operation button twice', () => {
+      instance.addNumber('6');
       instance.addMultiplication();
       instance.addMultiplication();
-      instance.addNumber(3);
+      instance.addNumber('3');
       instance.equal();
-      expect(instance.input()).to.equal(18);
-
-      instance.clear();
+      expect(instance.input()).to.equal("18");
+    });
 
-      instance.addNumber(9);
+    it('It should show correct result even if user pressed an divide operation button twice', () => {
+      instance.addNumber('9');
       instance.addDivision();
       instance.addDivision();
-      instance.addNumber(3);
+      instance.addNumber('3');
       instance.equal();
-      expect(instance.input()).to.equal(3);
+      expect(instance.input()).to.equal("3");
     });
 
     it('It should show "0" when divided by Zero', () => {
-      instance.addNumber(5);
+      instance.addNumber('5');
       instance.addDivision();
-      instance.addNumber(0);
+      instance.addNumber('0');
       instance.equal();
       expect(instance.input()).to.equal("0");
     });
 
     it('It should not add more than one zero if there was just zero on display', () => {
-      instance.addNumber(0);
-      instance.addNumber(0);
+      instance.addNumber('0');
+      instance.addNumber('0');
       expect(instance.input()).to.equal("0");
     });
 
-     it('Clicking zero after an operation should has the same effect of pressing clear button', () => {
-      instance.addNumber(1);
+    it('Clicking zero after an operation should has the same effect of pressing clear button', () => {
```

Comment:
This test is quite strange. I don't see how clicking zero has the same effect of pressing the clear button. I mean it looks the same, but it's not designed to be the same. It's similar to pressing any other number after an operation, it starts a new operation. I would delete this test because it can be confusing.